### PR TITLE
Fix >tally

### PR DESCRIPTION
**Pull request by keli5**
Looks like this fixes the new unbound local variable error when tallying a poll that lacks an image.

### CODE REVIEW

**Review by williammck — APPROVED** [review]

File: crimsobot/cogs/utilities.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ async def poll(self, ctx: commands.Context, *, poll_input: str) -> None:
         if len(question) == 0:
             question = 'QUICK POLL!'
 
+        attachment = None  # prevent unbound local var error
```
Comment (h-anjru) [inline crimsobot/cogs/utilities.py:44]: Why is attachment = None added again?

File: crimsobot/cogs/utilities.py [hunk 2]
```diff
@@ -104,7 +106,7 @@ async def polltally(self, ctx: commands.Context, poll_id: Optional[str] = None)
                     poll_found = True
                     descr = message.embeds[0].description
                     reactions = message.reactions
-
+                    image_url = None
```
Comment (h-anjru) [inline crimsobot/cogs/utilities.py:109]: This is the only line that needed to be added to fix the issue.

File: crimsobot/cogs/utilities.py (hunk 1 above)
Comment (williammck) [reply crimsobot/cogs/utilities.py:44]: oh right i didn't notice that... :thonk:

**Review by h-anjru — DISMISSED** [review]
Please remove the duplicate `attachment = None`.

**Review by williammck — APPROVED** [review]
